src/containers/AccountConnection.jsx

```diff
@@ -13,13 +13,14 @@ class AccountConnection extends Component {
   constructor (props, context) {
     super(props, context)
     this.store = this.context.store
+    this.state = {
+      account: this.props.existingAccount
```

Comment:
You're right. We can keep like this for now it's not really the goal of this PR so let's discuss about that later 👍